src/aws_client.erl

```diff
@@ -131,15 +141,29 @@ make_temporary_client_test() ->
                                        <<"some-token">>,
                                        <<"region">>)).
 
-make_local_client_test() ->
+make_local_client_3_test() ->
     ?assertEqual(#{access_key_id => <<"access-key-id">>,
 		   port => <<"8000">>,
 		   proto => <<"http">>,
 		   region => <<"local">>,
+                   endpoint => <<"localhost">>,
 		   secret_access_key => <<"secret-access-key">>,
 		   service => undefined},
 		 make_local_client(<<"access-key-id">>,
 				   <<"secret-access-key">>,
 				   <<"8000">>)).
 
+make_local_client_4_test() ->
+    ?assertEqual(#{access_key_id => <<"access-key-id">>,
+		   port => <<"8000">>,
+		   proto => <<"http">>,
+		   region => <<"local">>,
+                   endpoint => <<"endpoint">>,
```

Comment:
Tabs vs. spaces 😢

Reply:
Git blame points at https://github.com/aws-beam/aws-erlang/commit/8c0890e21144f4f9d04263f3a8bfc8676ab2d71f for introducing the tabs :-D I might get back to this module very soon so in that case I'll `untabify` it :+1: